webpack/ForemanTasks/Components/TasksDashboard/Components/ScheduledTasksCard/ScheduledTasksCard.test.js

```diff
@@ -0,0 +1,14 @@
+import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';
+import ScheduledTasksCard from './ScheduledTasksCard';
+
+const createRequiredProps = () => ({
+  scheduled: 1,
+});
+
+const fixtures = {
+  'render with minimal props': {},
+  'render with props': { ...createRequiredProps() },
```

Comment:
The are not actually required, would make more sense as
```js
'render with props': { scheduled: 1 },
```

Reply:
thanks